subprojects/plugins/src/main/java/org/gradle/api/plugins/jvm/JvmComponentDependencies.java

```diff
@@ -116,4 +119,28 @@ public interface JvmComponentDependencies {
      * @since 7.5
      */
     void annotationProcessor(Object dependencyNotation, Action<? super Dependency> configuration);
+
+    /**
+     * Declares a dependency on the test fixtures of a project.
+     * @param project the project upon which to add a test fixtures dependency
+     *
+     * @since 7.6
+     */
+    Dependency testFixtures(Project project);
+
+    /**
+     * Declares a dependency on the test fixtures of a project.
+     * @param projectDependency the project dependency for a project upon which to add a test fixtures dependency
+     *
+     * @since 7.6
+     */
+    Dependency testFixtures(ProjectDependency projectDependency);
```

Comment:
Are both of these methods needed?  I added the `ProjectDependency` one for consistency with `ModuleDependency` but aren't sure if build authors would be handling these objects directly.  Maybe in plugins?  IDK.

Reply:
You could probably just do `Object` and have one method.  The other work is going to replace this mostly.